archlinuxcn/lceda-pro/PKGBUILD

```diff
@@ -3,24 +3,24 @@
 
 pkgbase=lceda-pro
 pkgname=lceda-pro
-pkgver=2.1.42
+pkgver=2.1.44.1
 pkgrel=4
 pkgdesc="免费、专业、强大的国产PCB设计工具"
 arch=('x86_64' 'aarch64' 'loong64')
 url="https://pro.lceda.cn/"
-license=('custom')
+license=('LicenseRef-LCEDA-Proprietary')
 depends=(gtk3 nss alsa-lib electron libappindicator-gtk3 libnotify)
 install=${pkgname}.install
```

Comment:
```
lceda-pro E: Dependency python detected and not included (python modules ['os', 'sys', 'tarfile'] needed in files ['usr/lib/lceda-pro/resources/app/node_modules/sqlite3/deps/extract.py'])
lceda-pro W: Dependency bash detected and implicitly satisfied (programs ['sh'] needed in scripts ['usr/bin/lceda-pro', 'usr/lib/lceda-pro/resources/app/node_modules/sqlite3/tools/docker/architecture/linux-arm/run.sh', 'usr/lib/lceda-pro/resources/app/node_modules/sqlite3/tools/docker/architecture/linux-arm64/run.sh'])
lceda-pro W: Dependency hicolor-icon-theme detected and implicitly satisfied (needed for hicolor theme hierarchy)
lceda-pro W: Dependency gcc-libs detected and implicitly satisfied (libraries ['usr/lib/libstdc++.so.6', 'usr/lib/libgcc_s.so.1'] needed in files ['usr/lib/lceda-pro/resources/app/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node'])
lceda-pro W: Dependency coreutils detected and implicitly satisfied (programs ['echo'] needed in scripts ['usr/lib/lceda-pro/resources/app/node_modules/sqlite3/tools/docker/architecture/linux-arm/Dockerfile', 'usr/lib/lceda-pro/resources/app/node_modules/sqlite3/tools/docker/architecture/linux-arm64/Dockerfile', 'usr/lib/lceda-pro/resources/app/node_modules/sqlite3/Dockerfile'])
lceda-pro W: Dependency glibc detected and implicitly satisfied (libraries ['usr/lib/libm.so.6', 'usr/lib/libc.so.6', 'usr/lib/libpthread.so.0'] needed in files ['usr/lib/lceda-pro/resources/app/node_modules/sqlite3/lib/binding/napi-v3-linux-x64/node_sqlite3.node'])
lceda-pro W: Dependency included, but may not be needed ('gtk3')
lceda-pro W: Dependency included, but may not be needed ('nss')
lceda-pro W: Dependency included, but may not be needed ('alsa-lib')
lceda-pro W: Dependency included, but may not be needed ('electron')
lceda-pro W: Dependency included, but may not be needed ('libappindicator-gtk3')
lceda-pro W: Dependency included, but may not be needed ('libnotify')
```

Reply:
electron is needed; I have no idea for other dependencies, maybe ref #2981? @taotieren

Reply:
1. Can you make sure that the upstream dependency of electron is up to date? I recommend using the electron bundled with the upstream rather than the system electron.
2. I suggest changing it to the following dependency, but I don’t know this software and I can’t test if it works properly.

```diff
-depends=(gtk3 nss alsa-lib electron libappindicator-gtk3 libnotify)
+depends=(bash
+         hicolor-icon-theme
+         gcc-libs
+         coreutils
+         glibc)
```

Reply:
1. `lceda-pro/electron-version.txt` indicates the electron version is `19.1.0`. @taotieren said it is actually electron 21 https://github.com/archlinuxcn/repo/pull/2981#issuecomment-1333431517 and decided to use latest electron https://github.com/archlinuxcn/repo/pull/2981#issuecomment-1332015748. It works for now. I have no idea on whether changing it.

2. @taotieren I don't know why you added these dependencies, can you explain for us

Reply:
Have you verified that the software works properly without these dependencies?